sw/device/lib/dif/dif_kmac.h

```diff
@@ -374,7 +380,7 @@ typedef enum dif_kmac_error {
   /**
    * The entropy wait timer has expired.
    */
-  kDifErrorEntropyWaitTimerExpired,
+  kDifErrorEntropyWaitTimerExpired = 0x04000000,
```

Comment:
@ballifatih We talked offline and yeah it is not possible to add the assertion because it is not a enum on the register table. Could you add a comment saying it is coming from the RTL package?
But this is not blocking, we can enhance it in next PR ;)